node/src/gql/mutation/me/update-answer.ts

```diff
@@ -20,17 +20,20 @@ import {
 import { Status } from 'models/Answer';
 import { Mentor } from 'models/Mentor';
 import { User } from 'models/User';
+import requireEnv from 'utils/require-env';
 
 export interface AnswerUpdateInput {
   transcript: string;
   status: Status;
+  recordedAt: Date;
 }
 
-export const AnswerInputType = new GraphQLInputObjectType({
-  name: 'AnswerInputType',
+export const UpdateAnswerInputType = new GraphQLInputObjectType({
+  name: 'UpdateAnswerInputType',
   fields: () => ({
     transcript: { type: GraphQLString },
     status: { type: GraphQLString },
+    recordedAt: { type: GraphQLString },
```

Comment:
usually I would say client is not allowed to set timestamps, and the server should do that instead. Any reason we are enabling it here?

Reply:
mentor-upload is calling this too right? wouldn't we want mentor-upload to set recordedAt? or should there be two separate update answers.

Reply:
ok, so idea is to have distinct db fields for `recordedAt` vs `updatedAt` which might just be the transcription? Who is using this data?

Reply:
The client is using it to check whether or not they should fetch the recorded video for a mentor (using the mentor id and question id)

Reply:
We can leave for now if it's it makes the demo work, but when there is really video, the client should be basing that logic on the presence of actual media urls, probably something like

```json
{
  "transcript": "a b c",
  "media" [
    {
       "type": "video-mobile",
       "url": "..."
    },
    {
       "type": "video-web",
       "url": "..."
    }
  ]
}